src/css/imports/controlbar.less

```diff
@@ -31,12 +31,11 @@
   .jw-text-alt {
     display: none;
     position: absolute;
-    top: -1px;
-    bottom: 0;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
these changes should be applied on the skin level for inset, so that we don't impact existing customizations